modules/caddyhttp/encode/encode.go

```diff
@@ -195,6 +196,17 @@ func (enc *Encode) Match(rw *responseWriter) bool {
 	return enc.Matcher.Match(rw.statusCode, rw.Header())
 }
 
+func (rw *responseWriter) Flush() {
```

Comment:
I think this needs a proper godoc comment (i.e. `// Flush flushes the response. etc.`)